attributes/ms_dotnet4.rb

```diff
@@ -71,5 +71,15 @@
       hotfix_id = nt_version == 6.3 ? 'KB2934520' : 'KB2901982'
       default['ms_dotnet']['versions']['4.5.2']['package']['not_if']      = "wmic path Win32_QuickFixEngineering WHERE HotFixID='#{hotfix_id}' | FindStr #{hotfix_id}"
     end
+
+    if nt_version < 10.0
```

Comment:
Comments please. Could we have something a bit more tidy than nested if statements?